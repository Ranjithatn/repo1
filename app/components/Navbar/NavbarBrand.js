import React from "react";

const NavbarBrand = ({ logo, className, id }) => {
  return (
    <div
      className={`${"navbar-brand" + (className ? `${" " + className}` : "")}`}
      id={id}
    >
      <img src={logo} alt="logo" />
    </div>
  );
};

export default NavbarBrand;
