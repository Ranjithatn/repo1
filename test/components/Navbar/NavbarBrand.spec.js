import React from "react";
import toJson from "enzyme-to-json";
import NavbarBrand from "../../../app/components/Navbar/NavbarBrand";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });


describe('<NavbarBrand />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <NavbarBrand />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
  
});
