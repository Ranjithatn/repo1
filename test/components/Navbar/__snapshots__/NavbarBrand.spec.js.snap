// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarBrand /> should render without crashing 1`] = `
<NavbarBrand>
  <div
    className="navbar-brand"
  >
    <img
      alt="logo"
    />
  </div>
</NavbarBrand>
`;
